test/migrators/division/always_division.hrx

```diff
@@ -0,0 +1,37 @@
+<==> input/entrypoint.scss
+@function six-divided-by-three() {
+  @return 6px / 3px;
+}
+
+a {
+  b: (4px + 2px) / 3px;
+  c: 6px/3px + 1;
+  d: (6px / 3px);
+  e: 6px / (2px + 1px) / 2;
+  $f: 6px / 3px;
+
+  // * and % always return numbers, so we can assume division here
+  g: (3 * $x) / 4;
+  h: (3 / $x) % 4;
+  // < and similar operators always operate on numbers
+  i: 3 / $x < 1;
+}
+
+<==> output/entrypoint.scss
+@function six-divided-by-three() {
+  @return divide(6px, 3px);
+}
+
+a {
+  b: divide((4px + 2px), 3px);
```

Comment:
It would be nice, although not necessary, if we could get rid of parentheses here.